### PR TITLE
adds lastContact date to main list

### LINKED ISSUE #75: Add scope to fetch most recent comment date for a candidate
Depends on #73
This is how we will derive the 'last contacted date' for each candidate.

### PR DESCRIPTION
**Pull request by jshor**
Adds the most recent comment to each candidate (if any exist). Shows this comment's date on the list.

Fixes #75

### CODE REVIEW

File: server/models/CandidateModel.js [hunk 1]
```diff
@@ -36,6 +37,23 @@ var model = module.exports = db.define('candidate', {
   email: {
     type: Sql.STRING
   }
+
+}, {
+
+  defaultScope: {
```
Comment (jackofseattle) [inline server/models/CandidateModel.js]: This needs a comment to explain what it's doing. Someone that wasn't part of our conversation will be lost here.

File: server/models/_connection.js [hunk 2]
```diff
@@ -15,6 +15,6 @@ if (dbUrl) {
   module.exports = new Sequelize('sqlite://' + dbPath, undefined, undefined, {
     dialect: 'sqlite',
     storage: dbPath,
-    logging: process.env.DISABLE_SQL_LOGGING ? false : undefined
+    logging: process.env.DISABLE_SQL_LOGGING === 'true' ? true : false
```
Comment (jackofseattle) [inline server/models/_connection.js]: all comparisons result in a true or false already. You do not need the if else block here.

```js
process.env.DISABLE_SQL_LOGGING === 'true'
```
Will do just fine.

**Comment by jackofseattle** [comment]
A few small notes.  - Lets hold off on merging this until @dkoh fixes the db issues with staging though.

File: server/models/_connection.js (hunk 2 above)
Comment (jshor) [inline server/models/_connection.js]: sorry, I made this change but forgot to commit it

File: server/models/CandidateModel.js (hunk 1 above)
Comment (jshor) [inline server/models/CandidateModel.js]: added

File: server/models/_connection.js [hunk 3]
```diff
@@ -15,6 +15,6 @@ if (dbUrl) {
   module.exports = new Sequelize('sqlite://' + dbPath, undefined, undefined, {
     dialect: 'sqlite',
     storage: dbPath,
-    logging: process.env.DISABLE_SQL_LOGGING ? false : undefined
+    logging: process.env.DISABLE_SQL_LOGGING === 'true'
```
Comment (jackofseattle) [inline server/models/_connection.js]: I think this logic is inverted..

```js
process.env.DISABLE_SQL_LOGGING === 'true' // true - logging is now enabled
process.env.DISABLE_SQL_LOGGING !== 'true' // false, logging is now disabled
```

**Comment by jackofseattle** [comment]
Fix your merge conflicts and merge this in plz

**Pull request merged**